spec/sidekiq/scheduler_spec.rb

```diff
@@ -40,8 +40,29 @@
         'args'  => '/tmp'
       }
 
-      expect(Sidekiq::Client).to receive(:push).with(process_parameters(config))
+      expect(Sidekiq::Client).to receive(:push) do |opts|
+        expect(opts['class']).to eq(SomeWorker)
+        expect(opts['args']).to eq(['/tmp'])
```

Comment:
just making it explicit... I actually couldn't tell what was being tested.  Hope you don't mind :)